immichpy/cli/commands/users.py

```
@@ -188,7 +226,9 @@ def set_user_license(
         ..., "--activation-key", help="""Activation key"""
     ),
     license_key: str = typer.Option(
-        ..., "--license-key", help="""License key (format: IM(SV|CL)(-XXXX){8})"""
+        ...,
+        "--license-key",
+        help="""License key (format: /^IM(SV|CL)(-[\dA-Za-z]{4}){8}$/)""",
```

Ruff (0.15.20)

[warning] Invalid escape sequence: `\d`

Use a raw string literal

(W605)